django_datawatch/datawatch.py

```diff
@@ -94,12 +94,15 @@ def get_backend(self):
             self._backend = backend_module.Backend()
         return self._backend
 
-    def delete_results(self, sender, instance):
+    def delete_results(self, sender, instance, db_alias=None):
         from django_datawatch.models import Result
         for check_class in datawatch.get_checks_for_model(model=sender):
             check = check_class()
             identifier = check.get_identifier(instance)
-            Result.objects.filter(slug=check.slug, identifier=identifier).delete()
+            if db_alias:
```

Comment:
No need for a condition here, if `db_alias` is `None`, you can just pass that to `.using()` and it will go to the default DB.